tests/specs/openai/index.ts

```diff
@@ -0,0 +1,7 @@
+import { testSuite } from 'manten';
+
+export default testSuite(({ describe }) => {
+	describe('OpenAI', ({ runTestSuite }) => {
+		runTestSuite(import('./conventional-commits.js'));
+	});
+});
```

Comment:
The naming "OpenAI" of this file is confusing (the whole project is based on OpenAI).

Let's call it `diff-analysis`/`Diff analysis`.